storage/storage.go

```diff
@@ -941,7 +941,8 @@ func (o *ObjectHandle) ObjectName() string {
 
 // ObjectAttrsToUpdate is used to update the attributes of an object.
 // Only fields set to non-nil values will be updated.
-// Set a field to its zero value to delete it.
+// For all fields except CustomTime, set the field to its zero value to delete
+// it. CustomTime cannot be deleted or changed to an earlier time once set.
```

Comment:
For more information read: https://cloud.google.com/storage/docs/metadata#custom-time

Reply:
This is already present in the docs for ObjectAttrs, I don't think it needs to be repeated here (we don't go into detail for the other fields of this struct).

Reply:
Actually I guess it isn't there, I will add it in that location instead.